auth0/v3/management/connections.py

```diff
@@ -138,5 +138,4 @@ def delete_user_by_email(self, id, email):
         Returns:
             An empty dict.
         """
-        return self.client.delete(self._url(id) + '/users', params={'email': email})
-
+        return self.client.delete('{}/users'.format(self._url(id)), params={'email': email})
```

Comment:
This is confusing to read ... how about:

```
return self.client.delete( self._url( '{}/users'.format( id ) ) ...
```

Reply:
@joshcanhelp 
`return self.client.delete( self._url( '{}/users'.format( id ) ) ...` 
Actually 'id' is only expected in self._url(id). Above mentioned code passing both id+'/users'.
Only 'id' is expected in self._url() method.

For readability we can change this to 
`return self.client.delete('{_url}/users'.format(_url=self._url(id)), params={'email': email})`
or use the same.
Let me know your thought on this.

Reply:
Let's leave it as it was before:

```
self._url(id) + '/users'
```

I don't think the string formatting is adding anything here.

Reply:
Ok, left it as it was before.